defopt.py

```diff
@@ -312,8 +321,29 @@ def _parse_doc(func):
     return _Doc(doctext, tuples)
 
 
+def _itertext_ansi(node):
+    # Implementation modified from `ElementTree.Element.itertext`.
+    tag = node.tag
+    if (not isinstance(tag, (basestring if sys.version_info < (3,) else str))
```

Comment:
Sorry to keep getting you to change this, but this is a bit cluttered for my taste. Can you instead add a definition for basestring at the top with the rest of the compatibility stuff?